Plotting/composite_plots.py

```diff
@@ -88,26 +78,34 @@ def bin_derived_props_plots(fitspath, hbeta_bin = False, err_bars = False, revis
         ax1[1, 0].scatter(logM_avg[detect[ii]], log_threeBeta[detect[ii]], marker = '.', color = pt_color)
         ax1[1, 1].scatter(logM_avg[detect[ii]], logR[detect[ii]], marker = '.', color = pt_color)
     for ii in range(len(non_detect)):
-        if hbeta_bin == True:
+        if hbeta_bin:
             if non_detect[ii] % 2 == 0:
                 pt_color = 'pink'
             else:
                 pt_color = 'red'
-            ax1[1, 2].scatter(logM_avg[non_detect[ii]], logLHb_avg[non_detect[ii]], marker = '^', color = pt_color)
+            ax1[1, 2].scatter(logM_avg[non_detect[ii]], logLHb_avg[non_detect[ii]], marker = '^', 
+                              color = pt_color)
         else:
             pt_color = 'orange'
         ax1[0, 0].scatter(logM_avg[non_detect[ii]], logR23[non_detect[ii]], marker = '^', color = pt_color)
         ax1[0, 1].scatter(logM_avg[non_detect[ii]], logO32[non_detect[ii]], marker = '^', color = pt_color)
-        ax1[0, 2].scatter(logM_avg[non_detect[ii]], log_twoBeta[non_detect[ii]], marker = '^', color = pt_color)
-        ax1[1, 0].scatter(logM_avg[non_detect[ii]], log_threeBeta[non_detect[ii]], marker = '^', color = pt_color)
+        ax1[0, 2].scatter(logM_avg[non_detect[ii]], log_twoBeta[non_detect[ii]], marker = '^', 
+                          color = pt_color)
+        ax1[1, 0].scatter(logM_avg[non_detect[ii]], log_threeBeta[non_detect[ii]], marker = '^', 
+                          color = pt_color)
         ax1[1, 1].scatter(logM_avg[non_detect[ii]], logR[non_detect[ii]], marker = '^', color = pt_color)
     
-    ax1[0, 0].annotate('$R_{23}$', [0.05, 0.97], xycoords = 'axes fraction', va = 'top', ha = 'left', fontsize = '7')
-    ax1[0, 1].annotate('$O_{32}$', [0.05, 0.97], xycoords = 'axes fraction', va = 'top', ha = 'left', fontsize = '7')
-    ax1[0, 2].annotate('[OII]/H$\\beta$', [0.05, 0.97], xycoords = 'axes fraction', va = 'top', ha = 'left', fontsize = '7')
-    ax1[1, 0].annotate('[OIII]/H$\\beta$', [0.05, 0.97], xycoords = 'axes fraction', va = 'top', ha = 'left', fontsize = '7')
+    ax1[0, 0].annotate('$R_{23}$', [0.05, 0.97], xycoords = 'axes fraction', va = 'top', ha = 'left', 
+                       fontsize = '7')
+    ax1[0, 1].annotate('$O_{32}$', [0.05, 0.97], xycoords = 'axes fraction', va = 'top', ha = 'left', 
+                       fontsize = '7')
+    ax1[0, 2].annotate('[OII]/H$\\beta$', [0.05, 0.97], xycoords = 'axes fraction', va = 'top', ha = 'left', 
+                       fontsize = '7')
+    ax1[1, 0].annotate('[OIII]/H$\\beta$', [0.05, 0.97], xycoords = 'axes fraction', va = 'top', ha = 'left', 
+                       fontsize = '7')
```

Comment:
```suggestion
    ax1[0, 0].annotate('$R_{23}$', [0.05, 0.97], xycoords = 'axes fraction', va='top', ha='left', 
                       fontsize='7')
    ax1[0, 1].annotate('$O_{32}$', [0.05, 0.97], xycoords = 'axes fraction', va='top', ha = 'left', 
                       fontsize='7')
    ax1[0, 2].annotate('[OII]/H$\\beta$', [0.05, 0.97], xycoords='axes fraction', va='top', ha='left', 
                       fontsize='7')
    ax1[1, 0].annotate('[OIII]/H$\\beta$', [0.05, 0.97], xycoords='axes fraction', va='top', ha='left', 
                       fontsize='7')
```

Reply:
You might find that a for loop passing in a `zip()` of the matplotlib ax and the annotation string more simpler.